src/partials/login-tmpl.hbs

```diff
@@ -0,0 +1,24 @@
+
+<div class="login-section">
+    <div class="login-section_info">
+        <h2 class="login-section-heading">Login</h2>
```

Comment:
You can add H2 only if you have H1 above this on your page. You can make it H1 if it is the only Heading on page.

Reply:
changed